content/docs/guides/vercel-overview.md

```diff
@@ -1,59 +1,107 @@
 ---
-title: Neon and Vercel overview
-subtitle: Learn about different options for integrating Neon with Vercel
+title: Integrating Neon with Vercel
+subtitle: Choose the right connection path in seconds
 redirectFrom:
   - /docs/guides/vercel-postgres
 enableTableOfContents: true
-isDraft: false
-updatedOn: '2025-02-27T14:57:44.624Z'
+updatedOn: '2025-07-02T00:00:00.000Z'
 ---
 
-Neon supports different options for integrating Neon and Vercel, including a native integration that you can install from the Vercel Marketplace, a "previews integration" that creates a database branch with every pull request, and a manual setup option. If you're currently a Vercel Postgres user, you'll also find information below about the upcoming transition from Vercel Postgres to Neon.
+## Overview
 
-## Option 1: Vercel Native Integration
+This page helps you quickly choose the best Neon–Vercel integration for your project. Whether you're starting fresh or have existing infrastructure, we'll guide you to the right solution.
 
-This integration is intended for Vercel users who want to add Neon Postgres to their Vercel project as a [first-party native integration](https://vercel.com/docs/integrations/install-an-integration/product-integration). The integration creates a Neon Postgres account for you if you do not have one. You get access to Neon features and plans. **Billing is managed through Vercel**. The integration also supports automatic creation of a database branch with each Vercel preview deployment so that you can preview application and database changes together without impacting your production database.
+<Admonition type="tip" title="Quick Start">
+Most new users should start with the **Vercel-Managed Integration** for the simplest setup and unified billing. Existing Neon users can seamlessly connect with the **Neon-Managed Integration**.
```

Comment:
Maybe something like this:

Choose the Vercel-Managed Integration if you’re coming to Neon through Vercel and want unified billing through Vercel.
Choose the Neon-Managed Integration if you already have a Neon account or prefer to manage billing directly with Neon.

Reply:
Good idea, fixed.